lansweeper_app_for_splunk/default/data/ui/views/ls_assets.xml

```diff
@@ -62,14 +77,19 @@
         <fieldForLabel>State</fieldForLabel>
         <fieldForValue>State</fieldForValue>
         <search>
-          <query>`ls_assets_data` site_name=$tkn_site|s$ | rename "assetCustom.stateName" as State | dedup State</query>
+          <query>`ls_assets_data`   | dedup Statename</query>
           <earliest>-4h@h</earliest>
           <latest>now</latest>
         </search>
       </input>
       <table>
         <search>
-          <query>`ls_assets_data` site_name=$tkn_site|s$ "assetCustom.stateName"=$tkn_asset_state|s$ | dedup id | rename "asset.assetName" as Name, "asset.assetTypeName" as Type, "assetCustom.stateName" as State, "asset.userDomain" as Domain, "asset.assetGroups{}.groupName" as GroupName, "asset.ipAddress" as IPAddress, "asset.mac" as MacAddress, "asset.osCodeId" as OSVersion, "asset.userName" as User, "asset.firstSeen" as FirstSeen, "asset.lastSeen" as LastSeen, "asset.lastChanged" as LastChanged | table Name, Type, State, Domain, GroupName, IPAddress, MacAddress, OSVersion, User, FirstSeen, LastSeen, LastChanged</query>
+          <query>`ls_assets_data`  Statename=$tkn_asset_state|s$  
+| eval id=coalesce(AssetID,id) 
+| dedup id 
+| rename AssetName as Name, AssetTypename as Type, Statename as State, Userdomain as UserDomain, AssetGroup as GroupName, IPAddress as IPAddress, Mac as MacAddress, OScode as OSVersion, Username as User, Firstseen as FirstSeen, Lastseen as LastSeen, LastChanged as LastChanged 
```

Comment:
Let's add OS (coalesced one from SQL, for cloud, we'll coalesce from whatever we get) field after Asset Type field to see if it adds any more value to it.

Reply:
Added field OS